llvm/lib/CodeGen/BasicBlockSectionsProfileReader.cpp

```diff
@@ -61,8 +81,24 @@ BasicBlockSectionsProfileReader::getBBClusterInfoForFunction(
 // aliases. Basic block clusters are specified by 'c' and specify the cluster of
 // basic blocks, and the internal order in which they must be placed in the same
 // section.
+// This profile can also specify cloning paths which instruct the compiler to
+// clone basic blocks along a path. The cloned blocks are then specified in the
+// cluster information.
+// The following profile lists two cloning paths (starting with 'p') for
+// function bar and places the total 11 blocks within two clusters. Each cloned
```

Comment:
Perhaps add a ascii art of the CFG before and after cloning with the labels to illustrate the the prefix/cluster format?